doc/api/dns.md

```diff
@@ -159,6 +162,8 @@ Create a new resolver.
     each name server before giving up. **Default:** `4`
   * `maxTimeout` {integer} The max retry timeout, in milliseconds.
     **Default:** `0`, disabled.
+  * `rotate` {boolean} Perform round-robin selection of the nameservers for each resolution.
+    **Default:** `undefined`, depends on c-ares.
```

Comment:
Is this desired? Most users of Node.js are probably not even aware of c-ares. Also downstream @nodejs/distros rebuilders of Node.js may link to different versions of c-ares, so you may end up with variations of behaviour for the same version of Node.js depending on how you installed it.

Reply:
> Most users of Node.js are probably not even aware of c-ares

Agreed, we should probably not refer to that here.

Based on the commit message that added `ARES_OPT_NOROTATE` (https://git.chiebot.com:10000/Chiebot-Mirror/c-ares/commit/dd7a9bc89bf8d0d5c1888a7620680c9a013b9cc5), it sounds like both the library and the system configuration can influence the default, so we should say as much here, e.g.

```suggestion
    **Default:** `undefined`, depends on system configuration and Node.js build settings.
```